content/posts/irods-development-update-december-2023.md

```diff
@@ -0,0 +1,274 @@
+            - https://github.com/irods/irods_testing_environment/pull/180
+
+    - [iRODS GenQuery2](https://github.com/irods/irods_api_plugin_genquery2)
+        - packages for 4.2 and 4.3 can be built using the iRODS Development Environment
+            - use the plugin builder images
+
+    - [iRODS HTTP API](https://github.com/irods/irods_client_http_api)
+        - released 0.1.0
+        - investigating the idea of switching to an OAuth protected resource
+        - working on adding HTTPS support for well-known endpoint query
+        - working on OIDC JWT verification
+
+    - [Python iRODS Client (PRC)](https://github.com/irods/python-irodsclient)
+        - released 1.1.9
+        - added support for managing and querying quotas
+        - achieved iRODS 4.3 pam_password compatibility
+        - working on fix for data size / resource freespace compatibility
+        - working on TTL fix
+
+    - [Python Rule Engine Plugin](https://github.com/irods/irods_rule_engine_plugin_python)
+        - released for 4.3.1
+
+    - [Streaming S3 Resource Plugin](https://github.com/irods/irods_resource_plugin_s3)
+        - investigated issue with decoupled mode not being honored after a server redirect
+        - investigating issue with appending data via istream
+            - caused by iRODS doing a stat before the file exists in S3
+            - stat fails because the cache file has not be closed and stored in S3
+        - began process of absorbing libs3 into repository
+            - re-licensed plugin to LGPLv3/GPLv2
+            - will remove libs3 as an externals package
+
+    - [Globus Connector](https://github.com/irods/irods_client_globus_connector)
+        - ready for 4.3.1 release
+
+    - [iRODS S3 API](https://github.com/irods/irods_client_s3_api)
+        - released 0.1.0
+        - added/updated tests
+        - built a unittest docker framework
+        - resolved issues
+            - only include headers in SignedHeaders list for signature calculation
+            - copy operation now sets copy_options::overwrite_existing flag
+            - corrected GetBucketLocation operation
+            - fixed MinIO chunk transfer encoding issue by manually parsing the upload chunks
+        - additional features
+            - added support for setting the region
+            - implemented GetObjectLockConfiguration
+            - implemented HeadBucket
+        - removing Boost.URL submodule, will use externals package instead
+        - will be refactored to not use coroutines, to help the transition to libstdc++
+
+    - [Jargon](https://github.com/DICE-UNC/jargon)
+        - preparing release of 4.3.3.0
+        - added option to test framework that allows users to install other versions of iRODS 4.3
+        - updated FilePermissionEnum to support new permission levels
+        - added server version detection methods for iRODS 4.3
+        - StartupPack now sends the correct value for <option> element
```

Comment:
the option brackets don't render as you want - look at &lt and &gt.

Reply:
done